### ISSUE TITLE
Get current theme?

### ISSUE THREAD

**Issue by pixelrabbit** [etchteam/storybook-addon-css-variables-theme#28]
Is there a method to get the currently selected theme either in preview.js or within individual stories.  Maybe something like:
`console.log(parameters.cssVariables.currentTheme)`

Thanks.

**Comment by DanWebb** [comment]
@pixelrabbit Once https://github.com/etchteam/storybook-addon-css-variables-theme/pull/30 is released you'll be able to do this by watching the custom event on the `document`:

```typescript
  document.addEventListener(
    'storybookcssvariables:theme:change',
    (event: CustomEvent) => {
      console.info(`The theme changed to ${event?.detail?.theme}`);
    },
  );
```

**Comment by DanWebb** [comment]
Released in v1.4.0 🚀
